lib/suite.js

```diff
@@ -58,6 +58,7 @@ function Suite(title, parentContext) {
   this._slow = 75;
   this._bail = false;
   this._retries = -1;
+  this._deterministic = false;
```

Comment:
Bug? I see references to `this._randomized` below instead of `this._deterministic`.